lib/compat/wordpress-6.1/theme.json

```diff
@@ -246,6 +246,9 @@
 					"text": "#fff",
 					"background": "#32373c"
 				},
+				"spacing": {
+					"padding": "calc(0.667em + 2px) calc(1.333em + 2px)"
```

Comment:
I'm not sure what to do about this. In the theme I tested, the theme reset the padding for buttons. This padding is the one that used to be provided by the button block, but the search block uses a different padding. If we don't provide this then themes with heavy handed resets will lose button padding, but if we do provide this then the search block buttons will end up using this padding instead of the default specified in the block CSS. IMO this is a better compromise, but it will impact on the search button padding slightly.

Reply:
It is likely one of the smaller problems and I too agree that the second evil is lesser.